tools/packaging/kernel/build-kernel.sh

```diff
@@ -443,7 +454,7 @@ main() {
 				set -x
 				;;
 			e)
-				experimental_kernel="true"
+				build_type="experimental"
```

Comment:
@jcvenegas do we need to keep the `-e` flag if we can now specify experimental with `-b`?

Reply:
@cmaf I was about to remove it, but there are some scripts that still use them, I prefer to keep it as a legacy option.